pkg/gui/containers_panel.go

```diff
@@ -90,6 +94,37 @@ func (gui *Gui) handleContainerSelect(g *gocui.Gui, v *gocui.View) error {
 	return nil
 }
 
+func (gui *Gui) renderContainerEnv(container *commands.Container) error {
+	mainView := gui.getMainView()
+	mainView.Autoscroll = false
+	mainView.Wrap = gui.Config.UserConfig.Gui.WrapMainPanel
+	envVariablesList := [][]string{}
+	if len(container.Details.Config.Env) > 0 {
+		for _, env := range container.Details.Config.Env {
+			splitEnv := strings.Split(env, "=")
+			// if the value has = in it, lets say export "test=foo=bar" then split will result in the following
+			// {"test", "foo","bar"} hence join all the elements in the slice except the first one to get value
+			envVariablesList = append(envVariablesList,
+				[]string{
+					utils.ColoredString(splitEnv[0]+":", color.FgBlue),
```

Comment:
Blue is a little hard to see on some terminals, so I reserve it for things that aren't too important to see (like the `none` status of a service). Could we instead use green instead?

Reply:
Changed the color to blue